src/components/ChatSpinner.vue

```diff
@@ -14,19 +14,27 @@
 </template>
 
 <script>
-export default {
-  props: {
-    value: {
-      type: Boolean,
-      required: true
-    }
-  },
-  computed: {
-    className () {
+import { defineComponent, computed, defineProps } from 'vue'
+
+export default defineComponent({
+  setup () {
+    const className = computed(() => {
```

Comment:
const className= "chat-spinner"

Reply:
Хорошо. А почему надо computed свойства заменять на константы?

Reply:
Because it doesn't compute anything. In Vue2 it was not possible to use a variable or a const inside the `<template/>` without adding it ti the `data` or `computed` props. Now, in Vue3 you can, so the className no longer required to be reactive.